google-cloud-translate/CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Release History
 
+### 2.0.0 / 2019-10-28
+
+#### ⚠ BREAKING CHANGES
+
+* Add Translate V3 client
+  * Update google-cloud-translate to contain a generated v3 client
+    as well as the legacy hand-written v2 client.
+  * The following methods now return an instance of
+    Google::Cloud::Translate::V3::TranslationServiceClient:
+    * Google::Cloud#translate
+    * Google::Cloud.translate
+    * Google::Cloud::Translate.new
+  * To use the legacy v2 client specify the version when creating:
+    * Google::Cloud::Translate.new version: :v2
```

Comment:
Maybe: `v2_client = Google::Cloud::Translate.new version: :v2` to emphasize this is a code sample.

Reply:
Changed.